bigframes/session/__init__.py

```diff
@@ -213,6 +214,14 @@ def __hash__(self):
         # Stable hash needed to use in expression tree
         return hash(str(self._anonymous_dataset))
 
+    def clear_cache(self, table: str):
```

Comment:
Shall we let users define the snapshot time in read_gbq instead of exposing this function? What's this function used for?